client/components/domains/register-domain-step/index.jsx

```diff
@@ -343,6 +347,16 @@ class RegisterDomainStep extends React.Component {
 					/>
 				) }
 				{ this.content() }
+				{ isKrackenUI &&
+					! this.state.loadingResults && (
+						<Card
```

Comment:
Inspired by https://github.com/Automattic/wp-calypso/pull/23368, I've checked how navigating this new element by `Tab` looks like:
![2018-04-04 13_44_01](https://user-images.githubusercontent.com/3392497/38305738-3fa39502-380e-11e8-8204-6d20132a3f1c.gif)

Looks like it's missing focus state, even though the backing element for the link is a `button`? Probably best/easiest to try to replicate what `Use a domain I own` link uses? (unless it's some sort of hack ;))

Reply:
Definitely something wrong - I'll investigate this.